### PR TITLE
build(package.json): add package.json to project [MRF-83264]

### PR DESCRIPTION
**Pull request by oscard0m**
## 📝 Summary

- Add `package.json` file to project
- Add dummy test to be published as package
- Add Semantic Release GitHub Action to publish package

## ⛱ Motivation and Context

- Add configuration to publish end-to-end tests as a npm package
- Testing changes in a [beta](https://github.com/Marfeel/flowcards-playground/runs/2238358125?check_suite_focus=true) branch for the moment

## 📊 How Has This Been Tested?

- Running `npm pack` in local and checking published configuration
- Creating a branch called `beta` to test the Github Action

## 📚 References:

- https://docs.npmjs.com/cli/v7/configuring-npm/package-json#publishconfig
- https://docs.npmjs.com/cli/v7/commands/npm-pack
- https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "flowcards-playground",
```
Comment (oscard0m) [inline package.json]: What would be the name of the package?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "1.0.0",
```
Comment (oscard0m) [inline package.json]: Should I use `alpha`, `beta` and `0.x.x` until we agree is a stable version?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Flowcards Playground",
```
Comment (oscard0m) [inline package.json]: Is this enough or should describe something about End To End tests only?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Marfeel/flowcards-playground.git"
+  },
+  "author": "Marfeel",
+  "license": "ISC",
```
Comment (oscard0m) [inline package.json:16]: Not sure about the license here. This repo is supposed to be Open Source right?

File: package.json (hunk 1 above)
Comment (marcmrf) [reply package.json]: i would call it flowcards experiences

File: package.json (hunk 2 above)
Comment (marcmrf) [reply package.json]: it depends of how are you going to bump the versions, do you have any idea? most probably it's going to be always 1.0.1-bla-bla, so i don't thinks it's necessary to create an alpha. Not big deal.

Comment (oscard0m) [reply package.json]: Why `1.0.1-bla-bla`, there is no build process in this repository right? I was just thinking on using:
`npm version <update_type>` and `npm publish`

If we want to be fancy we can use https://github.com/semantic-release/semantic-release

File: package.json (hunk 1 above)
Comment (oscard0m) [reply package.json]:
```suggestion
  "name": "flowcards-experiences",
```

File: package.json (hunk 3 above)
Comment (oscard0m) [reply package.json]:
```suggestion
  "description": "Flowcards Experiences",
```

File: package.json (hunk 2 above)
Comment (oscard0m) [reply package.json]: Do not set a version. Semantic Release Github Action will manage it.

```suggestion
  "version": "0.0.0-development",
```

**Review by davidmohringmrf — APPROVED** [review]

**Pull request merged**